newsletters/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class department(models.Model):
```

Comment:
@dhruvhacks Is there a need of creating this model ? I think only faculties can upload the newsletters and you can find their department in the FacultyDetail table. Please let me know your thoughts over it.

Reply:
@RishabhJain2018 
Sir, while developing this feature, I created this model since every department will have multiple newsletters. Having ```department``` as an attribute within ```newsletters``` model will make redundant entries within it. A separate model will help more organised structure where all newsletters will be stored within one model linked to their respective departments.
Please tell if there is an alternative. :slightly_smiling_face: